src/session-search.test.ts

```
@@ -0,0 +1,80 @@
+import { describe, expect, it } from 'vitest';
+
+import {
+  extractSnippet,
+  findPromptMatch,
+  isMinorSession,
+  matchesAllWords,
+} from './session-search';
+
+describe('matchesAllWords', () => {
+  it('requires every word (AND semantics), matching across the combined text', () => {
+    const target = 'codev multi account /effort 我問一下 sessions 上限'.toLowerCase();
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)